client/src/components/FourOhFour/404.css

```diff
@@ -29,17 +29,8 @@
   text-align: left;
 }
 
-.quote-wrapper span {
-  font-size: 50px;
-  font-weight: 700;
-  color: var(--tertiary-color);
-  font-style: normal;
-  padding-inline-start: 15px;
-  padding-top: 5px;
-  position: absolute;
-  border-radius: 0;
-  top: 0;
-  left: 0;
+.quote-wrapper .quote::before {
+  content: open-quote;
```

Comment:
Maybe I am biased, but I like the small quote more 🤔.

This suggestion to have the same `font-weight` as before, and slightly bigger than `.quote`. it will look like this in the end

![image](https://user-images.githubusercontent.com/88248797/226428287-61c9baa1-8e45-4e99-8468-519954cb3bc0.png)


```suggestion
  content: open-quote;
  font-size: 25px;
  font-weight: 700;
```